### DISCUSSION TITLE
A option for unencrypted local automatic backups

### DISCUSSION THREAD

**Discussion by sebastianjacobs** [standardnotes/forum#2761] [Suggestions]
A additional option for the local automatic backups to store them unencrypted. Like the manual "decrypted" backups but for the automatic backups.

**Comment by JaspalSuri** [comment]
Hi @sebastianjacobs, thank you for the suggestion. Is there a reason why you would prefer the decrypted backups over the encrypted backups, given that you can use the included [Decrypt script](https://github.com/standardnotes/decrypt) (if you didn't want to use any of the apps to import and decrypt encrypted backups)?
